tests/test_stp_global.rb

```diff
@@ -0,0 +1,533 @@
+# Copyright (c) 2014-2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/stp_global'
+
+# TestX__CLASS_NAME__X - Minitest for X__CLASS_NAME__X node utility class
+class TestStpGlobal < CiscoTestCase
+  # TESTS
+
+  DEFAULT_NAME = 'default'
+
+  def setup
+    super
+    config 'no spanning-tree mode'
+    config 'system bridge-domain none' if n7k_platform?
+  end
+
+  def teardown
+    config 'no spanning-tree mode'
+    config 'system bridge-domain none' if n7k_platform?
+    super
+  end
+
+  def n7k_platform?
+    /N7/ =~ node.product_id
+  end
+
+  def n9k_platform?
+    /N(3|9)/ =~ node.product_id
+  end
+
+  def n6k_platform?
+    /N(5|6)/ =~ node.product_id
+  end
+
+  def create_stp_global(name=DEFAULT_NAME)
+    StpGlobal.new(name)
+  end
+
+  def test_bd_forward_time_change
+    skip('Platform does not support this property') if n6k_platform? ||
```

Comment:
We are moving away from using an `explicit` skip in the test case and instead allowing the `_exclude` to be the single source of truth.  Please make sure your testcases don't use an explicit `skip` and also make sure to do the following.

* Test each property like you normally would.  If the property is not supported on a platform, the following will automatically be generated.
  * https://github.com/cisco/cisco-network-node-utils/blob/develop/tests/basetest.rb#L97
* Make sure to test each property that has an exclude to make sure it get's coverage, otherwise a test case might get skipped early before all of the properties are tested.

```
Node under test:
  - name  - n6k-92
  - type  - N6K-C6001-64P
  - image - bootflash:///n6000-uk9-kickstart.7.3.0.N1.1.bin

TestStpGlobal#test_bd_priorities_change = 4.08 s = S

Finished in 4.085989s, 0.2447 runs/s, 0.0000 assertions/s.

  1) Skipped:
TestStpGlobal#test_bd_priorities_change [tests/test_stp_global.rb:104]:
Platform does not support this property

1 runs, 0 assertions, 0 failures, 0 errors, 1 skips
```